### PR TITLE
Anderson linting

### PR DESCRIPTION
**Pull request by anderson2981**
satisfy formatting?

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        steps:
+        -   uses: actions/checkout@v2
+        -
+            uses: actions/setup-python@v1
+            with:
+                # matches compat target in setup.py
+                python-version: '3.6'
+        -   name: "Main Script"
+            run: |
+                python -m pip install flake8
```
Comment (kaushikcfd) [inline .github/workflows/ci.yml:23]:
```suggestion
                python -m pip install flake8 flake8-quotes
```

File: nozzle.py [hunk 2]
```diff
@@ -276,32 +265,31 @@ def main(ctx_factory=cl.create_some_context, casename="nozzle", user_input_file=
 
     # param sanity check
     allowed_integrators = ["rk4", "euler", "lsrk54", "lsrk144"]
-    if(integrator not in allowed_integrators):
+    if (integrator not in allowed_integrators):
```
Comment (majosm) [inline nozzle.py:268]:
```suggestion
    if integrator not in allowed_integrators:
```

File: nozzle.py [hunk 3]
```diff
@@ -320,114 +308,131 @@ def main(ctx_factory=cl.create_some_context, casename="nozzle", user_input_file=
...
     # stagnation pressure 1.5e Pa
-    # 
-    # isentropic expansion based on the area ratios between the inlet (r=13e-3m) and the throat (r=6.3e-3)
+    #
+    # isentropic expansion based on the area ratios between the inlet (r=13e-3m)
+    # and the throat (r=6.3e-3)
     #
     # calculate the inlet Mach number from the area ratio
-    nozzleInletRadius = 13.e-3
+    nozzleInletRadius = 13.0e-3
```
Comment (majosm) [inline nozzle.py:324]: Camel case? 😱

File: nozzle.py [hunk 4]
```diff
@@ -436,23 +441,44 @@ def __call__(self, x_vec, *, time=0, eos, **kwargs):
...
+            )
+            temperature = getIsentropicTemperature(
+                mach=self._mach,
+                T0=T0,
+                gamma=gamma
+            )
             rho = pressure/temperature/gas_const
 
-            velocity = np.zeros(shape=(self._dim,)) 
+            velocity = np.zeros(shape=(self._dim, ))
```
Comment (majosm) [inline nozzle.py:456]: ?

File: nozzle.py [hunk 5]
```diff
@@ -552,37 +586,49 @@ def gen_sponge():
...
+                           dt=current_dt,
+                           cfl=current_cfl,
+                           eos=eos,
+                           t_final=t_final,
+                           constant_cfl=constant_cfl)
 
-    
     def sponge(cv, cv_ref, sigma):
-        return(sigma*(cv_ref-cv))
+        return (sigma * (cv_ref - cv))
```
Comment (majosm) [inline nozzle.py:615]:
```suggestion
        return sigma * (cv_ref - cv)
```

File: nozzle.py [hunk 6]
```diff
@@ -675,58 +731,81 @@ def my_checkpoint(step, t, dt, state, force=False):
...
+                        action="store_true",
+                        default=False,
+                        help="enable lazy evaluation [OFF]")
 
     args = parser.parse_args()
 
     # for writing output
     casename = "nozzle"
-    if(args.casename):
+    if (args.casename):
```
Comment (majosm) [inline nozzle.py:778]:
```suggestion
    if args.casename:
```

File: nozzle.py [hunk 7]
```diff
@@ -675,58 +731,81 @@ def my_checkpoint(step, t, dt, state, force=False):
...
         print(f"Custom casename {args.casename}")
-        casename = (args.casename).replace("'","")
+        casename = (args.casename).replace("'", "")
```
Comment (majosm) [inline nozzle.py:780]:
```suggestion
        casename = args.casename.replace("'", "")
```

File: nozzle.py [hunk 8]
```diff
@@ -675,58 +731,81 @@ def my_checkpoint(step, t, dt, state, force=False):
...
         print(f"Default casename {casename}")
 
-    snapshot_pattern="{casename}-{step:06d}-{rank:04d}.pkl"
-    restart_step=None
-    restart_name=None
-    if(args.restart_file):
+    snapshot_pattern = "{casename}-{step:06d}-{rank:04d}.pkl"
+    restart_step = None
+    restart_name = None
+    if (args.restart_file):
```
Comment (majosm) [inline nozzle.py:787]:
```suggestion
    if args.restart_file:
```

File: nozzle.py [hunk 9]
```diff
@@ -675,58 +731,81 @@ def my_checkpoint(step, t, dt, state, force=False):
...
         print(f"Restarting from file {args.restart_file}")
         file_path, file_name = os.path.split(args.restart_file)
         restart_step = int(file_name.split('-')[1])
-        restart_name = (file_name.split('-')[0]).replace("'","")
+        restart_name = (file_name.split('-')[0]).replace("'", "")
```
Comment (majosm) [inline nozzle.py:791]:
```suggestion
        restart_name = file_name.split('-')[0].replace("'", "")
```

File: nozzle.py [hunk 10]
```diff
@@ -675,58 +731,81 @@ def my_checkpoint(step, t, dt, state, force=False):
...
         print(f"step {restart_step}")
-        print(f"name {restart_name}") 
+        print(f"name {restart_name}")
 
-    input_file=None
-    if(args.input_file):
-        input_file = (args.input_file).replace("'","")
+    input_file = None
+    if (args.input_file):
+        input_file = (args.input_file).replace("'", "")
```
Comment (majosm) [inline nozzle.py:797]:
```suggestion
    if args.input_file:
        input_file = args.input_file.replace("'", "")
```

File: nozzle.py [hunk 11]
```diff
@@ -276,32 +265,31 @@ def main(ctx_factory=cl.create_some_context, casename="nozzle", user_input_file=
...
-        print(f'\tnviz = {nviz}')
-        print(f'\tnrestart = {nrestart}')
-        print(f'\tnhealth = {nhealth}')
-        print(f'\tnstatus = {nstatus}')
-        print(f'\tcurrent_dt = {current_dt}')
-        print(f'\tt_final = {t_final}')
-        print(f'\torder = {order}')
-        print(f"\tShock capturing parameters: alpha {alpha_sc}, s0 {s0_sc}, kappa {kappa_sc}")
+
+    if (rank == 0):
```
Comment (majosm) [inline nozzle.py:272]:
```suggestion
    if rank == 0:
```

File: nozzle.py [hunk 12]
```diff
@@ -276,32 +265,31 @@ def main(ctx_factory=cl.create_some_context, casename="nozzle", user_input_file=
...
+    viz_path = "viz_data/"
 
     dim = 3
-    exittol = .09
     current_cfl = 1.0
-    vel_init = np.zeros(shape=(dim,))
-    vel_inflow = np.zeros(shape=(dim,))
-    vel_outflow = np.zeros(shape=(dim,))
+    vel_inflow = np.zeros(shape=(dim, ))
+    vel_outflow = np.zeros(shape=(dim, ))
```
Comment (majosm) [inline nozzle.py:292]: ?

**Review by majosm — COMMENTED** [review]
Drive-by review.

File: nozzle.py [hunk 13]
```diff
@@ -1,4 +1,4 @@
-"""mirgecom driver for the Y0 demonstration.
+"""mirgecom driver for convering-diverging nozzle flow
```
Comment (MTCam) [inline nozzle.py:1]:
```suggestion
"""mirgecom driver for converging-diverging nozzle flow
```

File: nozzle.py (hunk 3 above)
Comment (MTCam) [reply nozzle.py:324]: I struggled with this too when first getting into CEESD (python general) style. 

My inkling was to replace all with underscores and lower case, but then the names are too long - easily running afoul of our preferred line-length limit, and very awkward looking expressions.

What I usually end up with is the minimally descriptive variable names that I hate to see when reading code, but have learned to live with it.  After accepting that (np == numpy), saving 3 letters but making it impossible to tell at a glance what `np` really is, I've lost my right to complain about descriptive naming.

`inlet_rad` works here?

File: nozzle.py (hunk 4 above)
Comment (MTCam) [reply nozzle.py:456]: Here's the parse error causing the linting error you were getting for line 1 of the driver, @anderson2981 .

File: nozzle.py (hunk 3 above)
Comment (anderson2981) [reply nozzle.py]: I removed the camel case (boo!)

**Pull request merged**